src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+const usersService = require('../services/users.service');
+const {
+  NOT_FOUND,
+  NO_CONTENT,
+  UNPROCESSABLE_CONTENT,
+  BAD_REQUEST,
+  CREATED,
+} = require('../constants/statuses.js');
+
+const get = (req, res) => {
+  const { query } = req;
+  const expensesFromServer = expensesService.getExpenses();
+
+  if (!Object.keys(query).length) {
+    res.send(expensesFromServer);
+
+    return;
+  }
+
+  const expensesByQuery = expensesService.getExpenseByQuery(query);
```

Comment:
Now you are getting expenses 2 times, it's not good
Try to refactor this and use only 1 function